advanced/advanced_numpy/index.rst

```diff
@@ -1148,7 +1148,16 @@ Generalized ufuncs
 
 * g-ufuncs are in Numpy already ...
 * new ones can be created with ``PyUFunc_FromFuncAndDataAndSignature``
-* ... but we don't ship with public g-ufuncs, except for testing, ATM
+* most linear-algebra functions are implemented as g-ufuncs to enable working
+  with stacked arrays
+
+>>> import numpy as np
```

Comment:
It would be good to indent this code block, so that it shows in the enumeration.

I also tend to prefer having explicit codeblock syntax, with "::" at the end of the line before.